packages/terra-show-hide/tests/jest/ShowHideFocuser.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ShowHideFocuser from '../../src/ShowHideFocuser';
+
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
```

Comment:
let's move this to a `beforeAll` function so we are setting and reseting them in the same place.

Reply:
For some reason I can't make it work with beforeAll. I looked for examples and everything I was able to find is in the way it is in this implementation. Can we make this change separately, outside of this PR as it is the way it's done terra-wide?

Reply:
That's fine by me!